docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3.8'
+
+services:
+  postgres:
+    image: postgres:latest
+    container_name: postgres_container
```

Comment:
It would be good to change the container name to one more descriptive or related to the project.

Reply:
Done, I put a descriptive name